webapp/src/css/inbox.less

```diff
@@ -248,10 +249,12 @@ body {
       }
 
       &.selected {
+        text-shadow: none;
         .button-label,
         .mm-icon:hover,
```

Comment:
To have the pills of the same size, add weight and height to .mm-icon
So when the tab stretches, the pill stays the same size

Reply:
Fixed it by adding the `width` and `height` to the `mm-icon` class not only when it's selected